docs/examples/random.md

```diff
@@ -0,0 +1,75 @@
+# opsdroid skill-random
+
+A skill for [opsdroid](https://github.com/opsdroid/opsdroid) to perform random events like flipping a coin or rolling a dice.
```

Comment:
We are talking to the reader here and telling them what we will cover in this page. Perhaps something like 'we will create a skill for performing random events...'

You can go into more detail about what the skill will do, why is it useful?